website/docs/d/iam_group.html.markdown

```diff
@@ -31,3 +31,15 @@ data "aws_iam_group" "example" {
 * `path` - The path to the group.
 
 * `group_id` - The stable and unique string identifying the group.
+
+* `members` - The member of group. See supported fields below.
```

Comment:
With Terraform 0.12, our documentation wording for explaining types is a little more important. 😄 Could you please note that this is a list of objects? e.g.

```suggestion
* `members` - List of objects containing group member information. See supported fields below.
```

Reply:
Thanks, i updated the docs